src/util/date/date.spec.ts

```diff
@@ -63,4 +68,50 @@ describe('Date utilities', () => {
       }
     )
   })
+
+  describe('Considering the formatISODateToParts function', () => {
```

Comment:
As discussed, I think that this test is useless as I have the feeling it is only testing the API, not a custom function. @fredericvilcot am I missing something?